BackEndDev/app.js

```diff
@@ -4,11 +4,9 @@ var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 
-var dotenv = require("dotenv")
-dotenv.config()
-
 var customerRouter = require('./routes/customers');
```

Comment:
this dotenv require is needed, make sure you pulled and merged the latest main into your branch